docs/configuration.rst

```diff
@@ -5,222 +5,47 @@ Configuration
-                1.fa.gz
-                1.fa.gz.fai
-                1.json
-                2.fa.gz
-                2.fa.gz.fai
-                2.json
-                # More references
-        datasets/
-            dataset1/
-                /variants/
-                    variantSet1/
-                        chr1.vcf.gz
-                        chr1.vcf.gz.tbi
-                        chr2.vcf.gz
-                        chr2.vcf.gz.tbi
-                        # More VCFs
-                    variantSet2/
-                        chr1.bcf
-                        chr1.bcf.csi
-                        chr2.bcf
-                        chr2.bcf.csi
-                        # More BCFs
-                /reads/
-                    sample1.bam
-                    sample1.bam.bai
-                    sample2.bam
-                    sample2.bam.bai
-                    # More BAMS
-
-.. note:: Any change to the data repository (using the repository manager or
-    otherwise) requires a restart of the server to be picked up by the
-    server.  The server does not detect changes in the data repository
-    while running.
-
-------------------
-Repository manager
-------------------
-
-The repository manager is a tool provided to abstract away the details of
-building a data repository behind a convenient command line interface.  It can
-be accessed via ``ga4gh_repo`` (or ``python repo_dev.py`` if developing).
-Following are descriptions of the commands that the repo manager exposes.
-
-All of the ``add-*`` commands take a ``--moveMode`` flag which specifies how
-to transfer the given file (or directory) into the data repository.  The
-options are ``move`` (moves the file from its original path to the new
-path), ``copy`` (copies the contents of the file into the data repository) and
-``link`` (creates a symlink in the data repository to the file).  The
-default is ``link``.
-
-Many of the ``add-*`` commands take additional flags to specify fields to be
-entered into the ``.json`` files that are created for the given file.
-Utilize the command line help for a particular command to get a list of
-these flags.
+The repository manager provides an administration interface to the the data
+repository. It can be accessed via ``ga4gh_repo`` (or ``python repo_dev.py`` if
+developing). Following are descriptions of the commands that the repo manager
+exposes.
 
 +++++++
 init
 +++++++
 
 Initializes a data repository at the path provided.  All of the other
-commands require a data repository path as an argument, so this will likely be
+commands require a data repository file as an argument, so this will likely be
 the first command you run.
 
 .. code-block:: bash
 
     $ ga4gh_repo init path/to/datarepo
 
+
 +++++++
-check
+verify
 +++++++
 
 Performs some consistency checks on the given data repository to ensure it is
 well-formed.
 
 .. code-block:: bash
 
-    $ ga4gh_repo check path/to/datarepo
+    $ ga4gh_repo verify path/to/datarepo
```

Comment:
Yeah, this is confusing. I've change it to ``/path/to/repo.db``; what do you think?